tests/data/test_study/extract_configs/simple_tsv_example2.py

```diff
@@ -34,10 +34,10 @@
     ),
     row_map(
         m=lambda row: row['storage_dir'] + '/' + row['bam_file_name'],
```

Comment:
these need to be lists now
```suggestion
        m=lambda row: [row['storage_dir'] + '/' + row['bam_file_name']],
```